engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -115,6 +115,9 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
       context stop self
       stay()
     case Event(JobStoreWriteFailure(_, t), _) =>
+      // This is moderately bad: If we can't write this result to the database, and then undergo a system restart, we'd need
+      // to re-run this job.
```

Comment:
The "need to re-run this job" mildly bothered me if an outside entity would never see it as complete, but if it is viewed as complete and gets re-run I don't think we can let that happen.